tests/e2e/api/run-newman-openai-integration.sh

```
@@ -0,0 +1,470 @@
+        local model="${BIFROST_MODEL:-gpt-4o}"
+        local embedding_model="${BIFROST_EMBEDDING_MODEL:-text-embedding-3-small}"
+        local speech_model="${BIFROST_SPEECH_MODEL:-tts-1}"
+        local transcription_model="${BIFROST_TRANSCRIPTION_MODEL:-whisper-1}"
+        local image_model="${BIFROST_IMAGE_MODEL:-dall-e-3}"
+        if [ -n "$ENV_FLAG" ]; then
+            cmd+=(-e "$ENVIRONMENT")
+        fi
+        cmd+=(--env-var "base_url=$base_url" --env-var "provider=$provider" --env-var "model=$model" --env-var "embedding_model=$embedding_model" --env-var "speech_model=$speech_model" --env-var "transcription_model=$transcription_model" --env-var "image_model=$image_model")
+    fi
+    [ -n "$FOLDER" ] && cmd+=(--folder "$FOLDER")
+    cmd+=(-r $REPORTERS)
```

Shellcheck (0.11.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)